src/coreclr/debug/daccess/cdac.cpp

```diff
@@ -88,7 +86,10 @@ CDAC::~CDAC()
         ::FreeLibrary(m_module);
 }
 
-IUnknown* CDAC::SosInterface()
+void CDAC::GetSosInterface(IUnknown** sos)
```

Comment:
Does this do an `AddRef()`? Typically, "outs" in COM do an `AddRef()`, where as when they are returned, non is taken.

Reply:
Yes, `Create*` sound better to me.